src/core/utils.jl

```diff
@@ -34,7 +34,16 @@ function annuity(years::Int, rate_escalation::Real, rate_discount::Real)
         i.e. it is a geometric sum of (1+rate_escalation)^n / (1+rate_discount)^n
         for n = 1, ..., years
     """
-    x = (1 + rate_escalation) / (1 + rate_discount)
+    return annuity_two_escalation_rates(years, rate_escalation, 0.0, rate_discount)
```

Comment:
This is very confusing and there seems to be no need for calling `annuity_two_escalation_rates` from `annuity` since the second escalation rate is always zero. Please revert the change in `annuity`.

Reply:
I did this refactor to eliminate duplicate code, but I'm fine to revert it if you find it confusing.